landoapi/projects.py

```diff
@@ -98,6 +114,35 @@ def get_checkin_project_phid(phabricator):
     return get_project_phid(CHECKIN_PROJ_SLUG, phabricator)
 
 
+@cache.cached(
+    key_prefix=TESTING_POLICY_PROJ_CACHE_KEY, timeout=DEFAULT_CACHE_KEY_TIMEOUT
+)
+def get_testing_policy_phid(phabricator):
+    """Return a phid for the project indicating testing policy.
+
+    Args:
```

Comment:
These `Args`/`Returns` comments should be type annotations.

Reply:
Fixed.